### DISCUSSION TITLE
Custom Field Permissions in 3.6

### DISCUSSION THREAD

**Discussion by barmalej1** [netbox-community/netbox#13987] [Help Wanted!] [answered]
I am on Netbox-docker v3.6. Crossposting here since I didn't get an answer in netbox-docker forum.

I recently upgraded from 3.4.5 to 3.6.1. I have a bunch of custom fields with custom field options. Previously, users without superadmin rights were able to see these values in dropdowns, but now when they press the arrow next to the custom field they get this pink popup saying that they don't have permissions to perform this action. I checked Admin -> Permissions. These 4 values:

extras.view_customfield
extras.view_customfieldchoice
extras.view_customfieldvalue
extras.view_customlink

are set to Can View for SuperAdmin, Users and Viewers.

Users have no problems to view other custom fields and values if there are no custom options involved. They also can update custom and other fields without any issues like before. Is this a bug or am I missing anything in the config?

**Comment by zmaster7** [comment, marked as answer]
In 3.6, custom field choices were moved into their own model, separate from the custom fields themselves (See #12988). My guess is that you need to add an additional view permission to account for this. 

![image](https://github.com/netbox-community/netbox/assets/46574676/818e63d8-c313-4813-877e-3fe5b8edf44e)

**Comment by barmalej1** [comment]
That did the trick, extras.view_customfieldvalue was there but Extras > Custom Field Chose Set wasn't selected. Thanks
